Libraries/AppDelegate/RCTAppDelegate.mm

```diff
@@ -79,11 +79,19 @@ - (void)applicationDidFinishLaunching:(NSNotification *)notification
   [self.window makeKeyAndVisible];
   return YES;
 #else // [macOS
-  self.window = [[NSWindow alloc] init];
-  NSViewController *rootViewController = [self createRootViewController];
+  NSRect frame = NSMakeRect(0,0,1024,768);
+  self.window = [[NSWindow alloc] initWithContentRect:NSZeroRect
+                                            styleMask:NSWindowStyleMaskTitled | NSWindowStyleMaskResizable | NSWindowStyleMaskClosable | NSWindowStyleMaskMiniaturizable
+                                              backing:NSBackingStoreBuffered
+                                                defer:NO];
+  self.window.title = self.moduleName;
+  self.window.autorecalculatesKeyViewLoop = YES;
+  NSViewController *rootViewController = [NSViewController new];
   rootViewController.view = rootView;
+  rootView.frame = frame;
   self.window.contentViewController = rootViewController;
   [self.window makeKeyAndOrderFront:self];
+  [self.window center];
```

Comment:
Taken from how we load RNTester for macOS

https://github.com/microsoft/react-native-macos/blob/04bf38c0c54595b0ac30db790d360719af228f6d/packages/rn-tester/RNTester/AppDelegate.mm#L129-L143